app/presenters/course_for_menu_presenter.rb

```diff
@@ -42,7 +42,12 @@ def to_h
       id: course.id,
       isFavorited: course.favorite_for_user?(@user) && @user.account.feature_enabled?(:unfavorite_course_from_dashboard),
       image: course.feature_enabled?(:course_card_images) ? course.image : nil,
-      position: @user.dashboard_positions[course.asset_string] || nil,
+      position: if @user.dashboard_positions[course.asset_string].present? == true
```

Comment:
A few small nitpicks:
- `== true` is redundant and unnecessary here because `.present?` already returns true or false
- I don't like seeing a full if/else/end block nested inside a hash definition like this. Just personal preference. I'd either extract this out into a variable above the hash definition, or extract `@user.dashboard_positions[course.asset_string]` into a shorter variable and then use an inline ternary expression (like on the previous line).

Also, the commented out line of code on 50 should just be removed.

These are all small minor things, and I have to import this into our internal system to run tests anyway, so I'll just fix them up when I do that. Overall the logic looks sound to me. We'll see what the tests think :)